hack/install-agent-rpm-ostree.sh.template

```diff
@@ -93,8 +102,8 @@ chmod 644 /etc/systemd/system/init-yggd.service
 
 systemctl enable init-yggd.service
 
-ln -s /usr/lib/systemd/system/podman.service /etc/systemd/system/default.target.wants/podman.service
-ln -s /usr/lib/systemd/system/podman.socket /etc/systemd/system/sockets.target.wants/podman.socket
+systemctl enable podman.service
+systemctl enable podman.socket
```

Comment:
question: why change only for podman and not for the other services below?

Reply:
Podman is already in there, part of the rhel-edge, but others ones cannot be enabled because does not exist yet!